src/main/java/seedu/address/logic/commands/delivery/DeliveryAddCommand.java

```diff
@@ -0,0 +1,209 @@
+package seedu.address.logic.commands.delivery;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_DELIVERY_DATE;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CUSTOMER_ID;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+
+import java.time.LocalDate;
+import java.util.Objects;
+import java.util.Optional;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ReadOnlyBook;
+import seedu.address.model.delivery.Delivery;
+import seedu.address.model.delivery.DeliveryDate;
+import seedu.address.model.delivery.DeliveryName;
+import seedu.address.model.delivery.DeliveryStatus;
+import seedu.address.model.delivery.OrderDate;
+import seedu.address.model.person.Customer;
+
+/**
+ * Adds a delivery to the delivery book.
+ */
+public class DeliveryAddCommand extends DeliveryCommand {
+
+    public static final String COMMAND_WORD = DeliveryCommand.COMMAND_WORD + " " + "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a Delivery to the HomeBoss database. "
+            + "Parameters: "
+            + "DELIVERY_NAME "
+            + PREFIX_CUSTOMER_ID + "CUSTOMER_ID "
+            + PREFIX_DATE + "DATE\n "
+            + "Example: " + COMMAND_WORD + " "
+            + "furniture "
+            + PREFIX_CUSTOMER_ID + "5 "
+            + PREFIX_DATE + "2023-12-03 ";
```

Comment:
Would it be better to have a whitespace between `PREFIX` and the subsequent string?